### PR TITLE
bulk transfers design (for existing diagrams)

### PR DESCRIPTION
**Pull request by elnyry**
1. Architecture changes to the bulk transfers diagram to include failure paths for bulk transfers at prepare, fulfil steps (at bulk level) and including failure path for fulfiling bulk individual transfers
2. Prepare handler updated to handle individual transfers in a bulk
3. Position handler 1.3.0, 1.3.1, 1.3.2, 1.3.3 updated to handle individual transfers in a bulk
4. Fulfil handler (fulfil, reject diagrams) updated to handle individual transfers in a bulk

### CODE REVIEW

**Comment by ggrg** [comment]
@elnyry, I've noticed something else, while working on the remaining bulk transfer diagrams. It applies to ALL sequence diagrams, not just the ones in this PR. All the hrefs from within the diagrams are not working any longer. Also, I'm not sure whether it is possible to reference svgs directly due to the http server setting to expose only .md files. For this purpose in diagram I'm currently busy with, I'm replacing all hrefs of type `https://github.com/mojaloop/docs/blob/master/**/*.svg` to `https://TODO-input-path-to/*.mdsvg`

File: mojaloop-technical-overview/central-ledger/assets/diagrams/sequence/seq-position-1.3.1-prepare.plantuml [hunk 1]
```diff
@@ -21,6 +21,7 @@
  * Georgi Georgiev <georgi.georgiev@modusbox.com>
  * Rajiv Mothilal <rajiv.mothilal@modusbox.com>
  * Miguel de Barros <miguel.debarros@modusbox.com>
+ * Samuel Kummary <sam@modusbox.com>
```
[the remark this answers is not in the document]
Comment (elnyry) [reply mojaloop-technical-overview/central-ledger/assets/diagrams/sequence/seq-position-1.3.1-prepare.plantuml]: Added, Thanks

File: mojaloop-technical-overview/central-ledger/assets/diagrams/sequence/seq-prepare-1.1.1.a.plantuml [hunk 2]
```diff
@@ -192,7 +229,12 @@ group Prepare Handler Consume
             deactivate POS_DAO
             
         end
-
+        note right of PREP_HANDLER #lightgrey
```
[the remark this answers is not in the document]
Comment (elnyry) [reply mojaloop-technical-overview/central-ledger/assets/diagrams/sequence/seq-prepare-1.1.1.a.plantuml]: Done

File: mojaloop-technical-overview/central-ledger/assets/diagrams/sequence/seq-prepare-1.1.1.a.plantuml [hunk 3]
```diff
@@ -124,58 +126,93 @@ group Prepare Handler Consume
...
-                                from: <transferMessage.payerFsp>,
-                                to: <transferMessage.payeeFsp>,
-                                type: application/json
-                                content: {
-                                    headers: <transferHeaders>,
-                                    payload: {
-                                        transferState: "COMMITTED",
-                                        fulfilment: "WLctttbu2HvTsa1XWvUoGRcQozHsqeu9Ahl2JW9Bsu8",
-                                        completedTimestamp: "2018-10-24T08:38:08.699-04:00"
+                        alt If type == 'prepare'
```
[the remark this answers is not in the document]
Comment (elnyry) [reply mojaloop-technical-overview/central-ledger/assets/diagrams/sequence/seq-prepare-1.1.1.a.plantuml]: I'd like to add a note here and instead add the actual detail on the bulk-prepare file, where it needs to be sent that way...

File: mojaloop-technical-overview/central-bulk-transfers/assets/diagrams/architecture/bulk-transfer-arch-flows.svg [hunk 4]
```diff
@@ -1,3 +1,3 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
```
[the remark this answers is not in the document]
Comment (elnyry) [reply mojaloop-technical-overview/central-bulk-transfers/assets/diagrams/architecture/bulk-transfer-arch-flows.svg]: Sure, lets do that after the DA Review with numbers close to final then

**Pull request merged**